config/routes.rb

```diff
@@ -285,9 +285,10 @@
         as: :user_two_factor_authentication # route name is used by two_factor_authentication gem
     get '/backup_code_refreshed' => 'users/backup_code_setup#refreshed'
     get '/backup_code_reminder' => 'users/backup_code_setup#reminder'
-    get '/backup_code_setup' => 'users/backup_code_setup#index'
+    get '/backup_code_setup' => 'users/backup_code_setup#create'
     patch '/backup_code_setup' => 'users/backup_code_setup#create', as: :backup_code_create
     patch '/backup_code_continue' => 'users/backup_code_setup#continue'
+    get '/backup_code_continue' => 'users/backup_code_setup#continue'
```

Comment:
I added that temporarily to try and fix a bug that's surfacing with this test: https://github.com/18F/identity-idp/blob/a2a2efdaeee38eee4d608abf6ddfdb114afe3534/spec/features/two_factor_authentication/multiple_mfa_sign_up_spec.rb#L87 
When I manually recreate the problem I see why it fails because instead of redirecting to /account it tries to do a GET at /backup_code_continue and throws a 404. The strange thing about that is when I do this on the main branch locally it behaves the same way. I can share a screencap if you're interested.

Reply:
As far as fixing this issue, I'm actually not sure we should be enforcing the reauthentication on this route anyways, since the backup codes are already associated with the account at this point. It's a little weird with how we [log analytics for MFA setup](https://github.com/18F/identity-idp/blob/3e3d1c33acfb1311583b953744dc956cd6928f4d/app/controllers/users/backup_code_setup_controller.rb#L39), but the `/backup_code_continue` route is effectively just a redirect, so I don't know why we'd expect it to have the reauthentication protection.

To fix the issue, I'd suggest we move the `multi_factor_auth_setup` analytics to where the backup codes are actually created ([`#create`](https://github.com/18F/identity-idp/blob/3e3d1c33acfb1311583b953744dc956cd6928f4d/app/controllers/users/backup_code_setup_controller.rb#L22)), [exempt the `:continue` action from reauthentication](https://github.com/18F/identity-idp/blob/3e3d1c33acfb1311583b953744dc956cd6928f4d/app/controllers/users/backup_code_setup_controller.rb#L14), and update the test accordingly (probably pick another MFA like phone setup that actually requires multiple steps to set up).

Reply:
> We should fix it... is it necessary to do it here? Are your changes causing that test to fail now?

Yes, exactly. I have a vague idea why it uncovers that bug. I'll take a look at your suggestion. When I'm using webauthn it's redirecting from /login/two_factor/webauthn

![Screenshot 2024-02-09 at 9 48 25 AM (2)](https://github.com/18F/identity-idp/assets/135744319/56ac32d3-7cfb-4de7-a0a1-422180c69de2)